pkg/resource/v1/certconfig/resource.go

```diff
@@ -11,40 +13,90 @@ import (
 const (
 	// Name is the identifier of the resource.
 	Name = "certconfigv1"
+
+	// Suggested maximum number of CertConfigs returned in one List() call to
+	// K8s API. Server may choose to not support this so this is not strict
+	// maximum.
+	listCertConfigLimit = 25
```

Comment:
Current default in protobuf for this is 7. Other defaults may vary. 25 is selected by gut feeling of what would be appropriate upper limit for single response. Currently we have 8 different `CertConfigs` defined.

Reply:
AFAIK godoc always starts with the name of the thing it is commenting on. 

```
listCertConfigLimit is the suggested maximum number ...
```

Reply:
Ah, of course. I was short sighted due to private variable.